tests/test.rs

```diff
@@ -115,8 +120,10 @@ fn nested_collection() {
 
     let s = r##"
         <project name="my_project">
-            <item name="hello1" source="world1.rs" />
-            <item name="hello2" source="world2.rs" />
+            <items>
```

Comment:
Unfortunately this form of sequence processing doesn't match how a lot of xml is written. I personally would prefer it this way though. Together with https://github.com/serde-rs/serde/issues/690 this would be the best solution.

Reply:
Yeah, it's a choice that I believe matches up better with the requirements of serde, but indeed it does not match all xml schemas. 

We could make it globally configurable in the serializer and deserializer, perhaps?

Reply:
It definitely doesn't match what I needed to parse and created serde-xml-rs for in the first place unfortunately :(

Reply:
Also, this representation is already supported with extra `struct` containing the collection, while adapting the other way is much harder.

Reply:
True. Using a serde annotation actually matches up with what I had to do to wrap primitives when serializing, so there's parity there.

I'm happy to revert this change. I see two options:
- Revert it and just use serde annotations to handle wrapped lists, as it was.
- Also add a flag to the deserializer, indicating whether all sequences should be deserialized as wrapped. I think this would be convenient for a lot of xml schemas that do wrap lists, and should cover most cases (on the assumption that schemas are likely to either always or never wrap list elements).

Any thoughts on these two options? I prefer the second one for convenience, but it does make the deserializer a bit more clunky from a design point of view.

Reply:
Oh you don't need to do anything on this end if https://github.com/serde-rs/serde/issues/690 is resolved on the serde end. In that case we'd use your serializer, fix the deserializer to match and just require the annotations in @RReverser's cases.

Reply:
Yeah if serde itself gets support for aggregating duplicate fields into vectors, that would completely cover my use case.